fs/march/march.go

```diff
@@ -178,6 +192,11 @@ func (m *March) Run() {
 	traversing.Wait()
 	close(in)
 	wg.Wait()
+
+	if errCount > 1 {
+		return fmt.Errorf("%+v (and %d others)", jobError, errCount-1)
```

Comment:
Can you make that something like `return errors.Wrapf(err, "march failed with %d errors: first error", errCount)` - rclone uses pkg/errors for error annotations and that will enable consumers to unwrap the error.